src/TileExpandable/README.md

```diff
@@ -0,0 +1,64 @@
+# Tile expandable
+To implement Tile expandable component into your project you'll need to import the TileExpandable, [TileExpandableHeader](#TileExpandableHeader) and the [TileExpandableContent](#TileExpandableContent):
+```jsx
+import TileExpandable from "@kiwicom/orbit-components/lib/TileExpandable";
+import TileExpandableHeader from "@kiwicom/orbit-components/lib/Tile/TileExpandableHeader";
+import TileExpandableContent from "@kiwicom/orbit-components/lib/Tile/TileExpandableContent";
+```
+After adding import into your project you can use it simply like:
+```jsx
+<TileExpandable>
+  <TileExpandableHeader icon={<Kiwicom />} title="Title" />
+  <TileExpandableContent>
+    Hello World!
+  </TileExpandableContent>
+</TileExpandable>
+```
+
+#### Props
+Table below contains all types of the props in TileContent component.
+
+| Name          | Type                  | Default         | Description                      |
+| :------------ | :---------------------| :-------------- | :------------------------------- |
+| **children**  | `React.Node`          |                 | The content of the TileExpandable.
+| shown         | `boolean`             | `false`         | if `true` the TileExpandable will have shown TileExpandableContent initially after render.
```

Comment:
Maybe I would formulate it like `If true, the TileExpandableContent will be shown by default`, so it's more clear what this prop is doing.